src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Main {
+    private static final FigureSupplier figureSupplier = new FigureSupplier();
```

Comment:
since you're using this instance in `main` method, you can declare `FigureSupplier figureSupplier = new FigureSupplier();` directly there - no problem with it :)

using such constants makes sense when dealing with methods, that are often invoked (which is not the case of `main` method), so that you don't create a new instance each time